homeassistant/components/glances/sensor.py

```diff
@@ -237,6 +237,12 @@ class GlancesSensorEntityDescription(
         icon="mdi:harddisk",
         state_class=SensorStateClass.MEASUREMENT,
     ),
+    ("uptime", "uptime"): GlancesSensorEntityDescription(
+        key="uptime",
+        type="uptime",
+        name_suffix="Uptime",
+        icon="mdi:clock-time-eight-outline",
+    ),
```

Comment:
Sensors like these must use the `timestamp` device class and return a valid `datetime` object.

Reply:
Changed that